apps/mobile/src/features/account/components/account-total-balance.tsx

```diff
@@ -28,12 +27,11 @@ export function AccountTotalBalance({ account }: AccountTotalBalanceProps) {
     accountIndex: account.accountIndex,
   });
   const isLoadingTotalBalance = totalBalance.state === 'loading';
-  const isErrorTotalBalance = totalBalance.state === 'error';
+
```

Comment:
Maybe we should just pass down `totalBalance` now? I'm not sure if it's better to call the `useAccountBalance` hook more often or else pass props down

Reply:
I always prefer getting state local to where it's needed rather than prop drilling unless perf is actually a concern.